apps/condo/domains/meter/schema/Meter.js

```diff
@@ -185,6 +186,22 @@ const Meter = new GQLListSchema('Meter', {
                 return addValidationError(`${DV_UNKNOWN_VERSION_ERROR}dv] Unknown \`dv\``)
             }
         },
+        afterChange: async ({ context, operation, originalInput, updatedItem }) => {
+            if (operation === 'update') {
+                const deletedMeterAt = get(originalInput, 'deletedAt')
+
+                if (deletedMeterAt) {
+                    const meterId = get(updatedItem, 'id')
```

Comment:
don't use get here! if you really don't have an ID it's better to throw an error rather than delete some objects by mistake